### ISSUE TITLE
How to host dashboard on service fabric mesh?

### ISSUE THREAD

**Issue by keshavraja** [OrleansContrib/OrleansDashboard#237]
I am hosting my orleans solution on service fabric mesh, So how do i configure this, i am able to run this locally but once i push code to service fabric mesh.. it is not working.

**Comment by richorama** [comment]
Hi @keshavraja,

I'm afraid this is not a scenario I have tried personally, but I know people do run this configuration.

Would you mind expanding on the problem?

How is it not working? Is port 8080 not responding when attempting to browse to it?

Is there any information in the log? The dashboard will report problems at startup.

Is the port publicly exposed in service fabric, and Azure?

What version of Orleans / OrleansDashboard are you using?

**Comment by keshavraja** [comment]
Hello @richorama 
I am using OrleansDashborad 2.3.0.

Scenario is,
I have created a Service Fabric Mesh Solution in which I have added Silo as Service fabric service.
port 80 i have used for another service in project, In UseDashborad I have used 85 port number
 .UseDashboard(options =>
                {
                    options.Username = "USERNAME";
                    options.Password = "PASSWORD";
                    options.Host = "*";
                    options.Port = 85;
                    options.HostSelf = true;
                    options.CounterUpdateIntervalMs = 1000;
                })

After deploying this to azure service fabric mesh and I try to open "https://ipaddress:85" it should open orleans dashboard but it is not.

I am not sure how do I expose 85 port on service fabric mesh.
